pubnub/endpoints/fetch_messages.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, pubnub):
         self._count = None
         self._include_meta = None
         self._include_message_actions = None
+        self._include_message_type = None
```

Comment:
I don't think we have it in Kotlin SDK :/

Reply:
i've included all fields that were required to completely mimic the behavior of REST endpoint